artemis-web/src/main/java/org/apache/activemq/artemis/component/WebServerComponent.java

```diff
@@ -170,4 +173,15 @@ private WebAppContext deployWar(String url, String warFile, Path warDirectory) t
       handlers.addHandler(webapp);
       return webapp;
    }
+
+   @Override
+   public void stop() throws Exception {
```

Comment:
please, just for completeness.. call stop(true); here (after the method name change I asked you)